drghs-worker/maintnerd/main.go

```diff
@@ -99,6 +107,33 @@ func main() {
 		log.Fatal(err)
 	}
 
+	cfg := profiler.Config{
+		Service:        fmt.Sprintf("maintnerd-%v-%v", *owner, *repo),
+		ServiceVersion: "0.0.1",
+		ProjectID:      *projectID,
+
+		// For OpenCensus users:
+		// To see Profiler agent spans in APM backend,
+		// set EnableOCTelemetry to true
+		//EnableOCTelemetry: true,
+	}
+
+	if err := profiler.Start(cfg); err != nil {
```

Comment:
Is this going to profile every single request? What's the performance cost?

Reply:
This is probabalistic. I blasted the dev service with ~6k requests and only got 30 ish profiles.